backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -146,6 +147,146 @@ public static IQueryable<Entity.Views.Property> GenerateQuery(this PimsContext c
             else
                 query = query.OrderBy(p => p.AgencyCode).ThenBy(p => p.PID).ThenBy(p => p.PIN).ThenBy(p => p.PropertyTypeId);
 
+            return query;
+        }
+
+
+        /// <summary>
+        /// Generate a query for the specified 'filter'.
+        /// Returns all properties, even properties that don't belong to the user's agency or sub-agencies.
+        /// The results of this query must be 'cleaned' so that only appropriate data is returned to API.
+        /// </summary>
+        /// <param name="context"></param>
+        /// <param name="user"></param>
+        /// <param name="filter"></param>
+        /// <returns></returns>
+        public static IQueryable<Entity.Views.Property> GenerateAllPropertyQuery(this PimsContext context, ClaimsPrincipal user, Entity.Models.AllPropertyFilter filter)
```

Comment:
A lot of this is duplicated from the other queries.  Tech-debt to extract duplication in an extension method at some point.